articles/200_migration_guide_from_ros1.md

```diff
@@ -145,6 +199,34 @@ Shared pointer types are provided as typedefs within the message structs: `my_in
 
 For more details please see the article about the [generated C++ interfaces](http://design.ros2.org/articles/generated_interfaces_cpp.html).
 
+The migration requires includes to change by:
+
+* insert the subfolder `msg` between the package name and message datatype
+* Change the included filename from CamelCase to underscore separation
+* Change from `*.h` to `*.hpp`
+
+The migration requires code to insert the `msg` namespace into all instances.
```

Comment:
This might be easier to understand with an example, like this:

```
// ROS 1 style is in comments, ROS 2 follows, uncommented.
// # include <geometry_msgs/PointStamped.h>
# include <geometry_msgs/msg/point_stamped.hpp>

// geometry_msgs::PointStamped point_stamped;
geometry_msgs::msg::PointStamped point_stamped;
```